modules/books.js

```diff
@@ -0,0 +1,70 @@
+export const mainContainer = document.querySelector('#books-container');
+class Books {
+  constructor() {
+    this.books = [];
+
+    if (localStorage.getItem('library')) {
+      // Re-assign the books array to the localStorage library
+      this.books = JSON.parse(localStorage.getItem('library'));
+    }
+  }
+
+  // A method to save the info in the local storage
+  storage(book) {
+    this.books.push(book);
+    localStorage.setItem('library', JSON.stringify(this.books));
+  }
```

Comment:
Kindly refactor your code to make use of `es6` arrow function as this is a requirement in this milestone👍.

```suggestion
  storage = (book) => {
    this.books.push(book);
    localStorage.setItem('library', JSON.stringify(this.books));
  }

```